porcupine/tabs.py

```diff
@@ -528,7 +528,7 @@ class FileTab(Tab):
         Things like :source:`the line number plugin <porcupine/plugins/linenumbers.py>` use this attribute.
 
     .. attribute:: path
-        :type: pathlib.Path | None
+        :type: Path | None
```

Comment:
I don't think the types display very well anyway, but let's not make it any worse than it is:

```suggestion
        :type: pathlib.Path | None
```